dev-client/src/screens/SiteScreen/SiteScreen.tsx

```diff
@@ -63,31 +61,42 @@ const LocationDetail = ({label, value}: {label: string; value: string}) => (
 
 type LocationPredictionProps = {
   label: string;
-  prediction: string;
-  confidence: string;
+  soilName: string;
+  ecologicalSiteName: string;
 };
 
 const LocationPrediction = ({
   label,
-  prediction,
-  confidence,
+  soilName,
+  ecologicalSiteName,
 }: LocationPredictionProps) => {
   const {t} = useTranslation();
 
   return (
-    <Column backgroundColor="primary.main" alignItems="center" py="18px">
-      <Text variant="body1" color="primary.contrast" bold>
+    <Column
+      backgroundColor="background.secondary"
+      alignItems="flex-start"
+      py="18px"
+      pl="16px">
+      <Text variant="body1" color="primary.lighter" bold>
+        <Box mr={15}>
+          <StackedBarChart />
+        </Box>
```

Comment:
@shrouxm Can I apply the margin to the SVG directly?

```js
<StackedBarChart marginRight={15} />
```
works, but gives a type error:
```
Property 'marginRight' does not exist on type 'IntrinsicAttributes & SvgProps'.
```

Reply:
Issue: On Android, I'm seeing this with your changes:
![Screenshot 2024-04-26 at 12 58 50 PM](https://github.com/techmatters/terraso-mobile-client/assets/5590815/50a4d0d0-209c-4d79-ac39-d3baa68bac45)

I suspect it's another one of these weird "Non-text components inside a <Text> component on Android" things. This should work though: 

`<Row>
    <Box bold mr={15}>
      <StackedBarChart />
    </Box>
    <Text variant="body1" color="primary.lighter" bold>
      {label}
    </Text>
</Row>`

Reply:
I've applied your suggestion.